server/config.go

```diff
@@ -41,6 +60,9 @@ type MatterwickConfig struct {
 	MattermostWebhookURL    string
 	MattermostWebhookFooter string
 
+	KubeClusterName   string
+	KubeClusterRegion string
```

Comment:
dont see those config in the config.json

Reply:
Added them in my gitignored copy but not the template. Good catch.